### PR TITLE
Clean up and simplify scripts and metadata.

### PR DESCRIPTION
**Pull request by LyonesGamer**
Clean up and improve the gitignore, travis file, readme, migrations, package.json, and scripts, as well as add a changelog and license file.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+v1.0.0
+~~~~~~
+
+* Initial release of TimeSync-Node!
```
Comment (ElijahCaine) [inline CHANGELOG.md]: Let's include a little blurb about the initial release, including something like `for a complete feature-list go to whatever.writethedocs.org/v1/

File: README.md [hunk 2]
```diff
@@ -86,33 +93,32 @@ your curl request:
...
 
 Your output should look something like the above.
 
-If you would like to test the Postgres database, but don't want to setup a
+If you would like to test the PostgreSQL database, but don't want to setup a
 whole Postgres database, the command ``npm run test_pg_docker`` will spin up a
 docker container and run the tests.
 
-The postgres container will continue to run until you manually kill it
+The Postgres container will continue to run until you manually kill it
```
Comment (ElijahCaine) [inline README.md]: `PostgreSQL`

File: README.md [hunk 3]
```diff
@@ -133,6 +140,10 @@ To run migrations on a particular backend, run:
 $ NODE_ENV=backend npm run migrations
 ```
 
+Valid values of NODE_ENV are ``mocha`` (Postgres), ``mocha_sqlite`` (SQLite),
+``development`` (SQLite), ``development_pg`` (Postgres), ``production_pg``
+(Postgres), ``production_mysql`` (MySQL), and ``production_sqlite`` (SQLite).
```
Comment (ElijahCaine) [inline README.md]: `PostgreSQL`

**Comment by ElijahCaine** [comment]
1. Add a blurb to the changelog either listing features, linking to docs, or both.
2. It's `Postgres` or `PostgreSQL`. We should be consistent.

**Comment by ElijahCaine** [comment]
:+1: awesomesauce

**Comment by athai** [comment]
Looks good. +1

**Comment by ElijahCaine** [comment]
@LyonesGamer Let's merge this

**Pull request merged**